api/test/integration/test_cluster_endpoints.tavern.yaml

```diff
@@ -756,18 +757,15 @@ stages:
       json:
         error: !anyint
```

Comment:
Review all error values in this file to indicate the correct ones instead of `!anyint`